### PR TITLE
Update infrastructure page

### PR DESCRIPTION
**Pull request by cherifGsoul**
For #5417 

Remove `can-util` modules and `can-dom-event`.

### CODE REVIEW

File: docs/can-canjs/can-infrastructure.md [hunk 1]
```diff
@@ -38,8 +37,22 @@ me.addEventListener("name", function(){ /* ... */ });
 me.dispatch("name");
 ```
 
-[can-event/batch/batch] adds event batching abilities to the [can-event] event system.
-[can-event/async/async] adds asynchronous batched event dispatching to the [can-event] event system.
+## can-queues
+
+A light weight queue system for scheduling tasks, it runs tasks in one of the following queues:
+
+1. [can-queues.notifyQueue] - Tasks that notify observables "deriving" observables that a source value has changed.
```
Comment (phillipskevin) [inline docs/can-canjs/can-infrastructure.md]: I know this is copied from the `can-queues` docs, but

> Tasks that notify observables "deriving" observables that a source value has changed.

...doesn't really make sense. How about?

> Tasks that notify "deriving" observables that a source value has changed.

File: docs/can-canjs/can-infrastructure.md [hunk 2]
```diff
@@ -50,8 +63,7 @@ The following makes the `Person` type’s `getName()` observable:
 
 ```js
 import Observation from 'can-observation';
-import canEvent from 'can-event';
-import assign from 'can-util/js/assign/assign';
+import mixinMapBindings from 'can-event-queues/map/map';
```
Comment (phillipskevin) [inline docs/can-canjs/can-infrastructure.md]: Does this need to be mixed in for the code below to work?

File: docs/can-canjs/can-infrastructure.md (hunk 1 above)
Comment (cherifGsoul) [reply docs/can-canjs/can-infrastructure.md]: Fixed!

File: docs/can-canjs/can-infrastructure.md [hunk 3]
```diff
@@ -12,22 +12,21 @@ be used by applications directly.
 
 Let’s explore what’s available.
 
-## can-event
+## can-event-queues
```
[the remark this answers is not in the document]
Comment (cherifGsoul) [reply docs/can-canjs/can-infrastructure.md]: Fixed!

File: docs/can-canjs/can-infrastructure.md (hunk 2 above)
Comment (cherifGsoul) [reply docs/can-canjs/can-infrastructure.md]: Updated!

**Pull request merged**